packages/webhooks/lib/sync.ts

```diff
@@ -63,7 +65,7 @@ export const sendSync = async ({
             integration: { id: providerConfig.id!, name: providerConfig.unique_key, provider: providerConfig.provider },
             connection: { id: connection.id, name: connection.connection_id },
             syncConfig: { id: syncConfig.id, name: syncConfig.sync_name },
-            meta: { scriptVersion: syncConfig.version }
```

Comment:
Should we put that in syncConfig and index it so we can filter logs with it?
I'm sure it would be useful in sync run too